### PR TITLE
Fix luacheck issues

### PR DESCRIPTION
**Pull request by streetturtle**

### CODE REVIEW

File: mpris-widget/init.lua [hunk 1]
```diff
@@ -68,7 +67,7 @@ function playerctl:watch(timeout, callback, widget)
             metadata.year = string.sub(words[8], 0, 4)
         end
 
-        callback(widget, metadata)
+        callback(cb_widget, metadata)
     end
 
     _, self.timer = awful.widget.watch(cmd, timeout, cb, widget)
```
Comment (streetturtle) [inline mpris-widget/init.lua:73]: Not sure how to fix this one, I guess I need help 😄 

> mpris-widget/init.lua:73:5: setting non-standard global variable _

Comment (Aire-One) [reply mpris-widget/init.lua:73]: I guess declaring a `local _` before this line, or assigning `self.timer = awful...()[2]` can do the trick.

However, it seems the code here wants to discard the widget and only use the timer from `awful.widget.watch`. Maybe the line should be replaced by `gears.timer`?

Comment (Ryuno-Ki) [reply mpris-widget/init.lua:73]:
```suggestion
    local _, timer = awful.widget.watch(cmd, timeout, cb, widget)
    self.timer = timer
```

It ain't pretty but gets the job done.

Comment (streetturtle) [reply mpris-widget/init.lua]: Thank you!

**Pull request merged**